src/test/resources/TestNG.xml

```diff
@@ -1,23 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 
-<suite name="TestNG">
-<parameter name="BaseURL" value="https://bbb.testpro.io/"/>
+<suite name="TestNG" parallel="methods" thread-count="2">
+
+    <parameter name="BaseURL" value="https://bbb.testpro.io/"/>
+
     <test name="Example Test" preserve-order="false">
         <classes>
             <class name="LoginTests"/>
-            <class name="LoginPage"/>
-            <class name="BasePage"/>
             <class name="BaseTest"/>
-            <class name="HomePage"/>
-            <class name="HomeWork17"/>
-            <class name="HomeWork18"/>
-            <class name="HomeWork19"/>
-            <class name="HomeWork20"/>
-            <class name="HomeWork21"/>
-            <class name="HomeWork22"/>
-            <class name="HomeWork23"/>
-            <class name="HomeWork24"/>
+
```

Comment:
I'll try to help you, here's an example of parallel testing in 4 threads on a local machine:



_Download_

1) Code with Russian comments https://drive.google.com/file/d/1dQjnroe0pIiIqll_dT8rTwgAO5cekj4W/view?usp=share_link


2) Code with English comments https://drive.google.com/file/d/1H6MUQcisY_XQ2kn47AhrGU5TBVxli1ae/view?usp=share_link


**To run the tests, open the README file in IntelliJ IDEA**


![image](https://github.com/Test-Pro-Bootcamp/A45-QA-Automation-Framework/assets/104761482/3c7532eb-65f0-4854-823b-be41ba43622a)

**click**  ![image](https://github.com/Test-Pro-Bootcamp/A45-QA-Automation-Framework/assets/104761482/7313f1bd-6aba-4a26-8f83-dd510a432c8c)

_PS:  To unzip a ZIP file on a Mac_ https://www.winzip.com/en/learn/file-formats/zip/mac/